Sources/MapboxMaps/Gestures/GestureHandlers/PinchGestureHandler.swift

```diff
@@ -113,12 +110,13 @@ internal final class PinchGestureHandler: GestureHandler, PinchGestureHandlerPro
             delegate?.gestureBegan(for: .pinch)
         }
     }
+}
 
-    private func pinchAngle(with gestureRecognizer: UIPinchGestureRecognizer) -> CGFloat {
-        // we guard for this at the call site
-        let view = gestureRecognizer.view!
-        let pinchPoint0 = gestureRecognizer.location(ofTouch: 0, in: view)
-        let pinchPoint1 = gestureRecognizer.location(ofTouch: 1, in: view)
-        return atan2(pinchPoint1.y - pinchPoint0.y, pinchPoint1.x - pinchPoint0.x)
+extension PinchGestureHandler: UIGestureRecognizerDelegate {
+    internal func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer,
```

Comment:
Should we change the method name to isXX as it returns a Bool value?

Reply:
The `gestureRecognizer(_:shouldRecognizeSimultaneouslyWith:)` is a part of API provided by `UIKit`, we can't really change anything there.